source/services/reddit.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Reddit service provider
+ *
+ * Notes:
+ * 1) How are shares counted?
+ * The number of up votes minus the number of down votes for a specific URL submitted on Reddit
+ * `children[0].data.score` - contain pre calculated value
+ *
+ * 2) An API request may fail because of CORB so we need to use JSONP to make it work
```

Comment:
Do we still need these notes? Everything's in the code now, isn't it?

Reply:
You are right. There is no longer any sense in this comment. I will delete this comment